ch05.adoc

```diff
@@ -183,13 +183,16 @@ _This section has been superseded by the treatment of time series as a type of d
 [[grid-mappings-and-projections, Section 5.6, "Horizontal Coordinate Reference Systems, Grid Mappings, and Projections"]]
 === Horizontal Coordinate Reference Systems, Grid Mappings, and Projections
 
-When the coordinate variables for a horizontal grid are not longitude and latitude, it is required
-that the true latitude and longitude coordinates be supplied via the **`coordinates`** attribute.  A
-__grid mapping variable__ is required if, in addition, it is desired to describe the mapping between the
-given coordinate variables and the true latitude and longitude coordinates, or to describe the
+A __grid mapping variable__ may be referenced by a data variable, to describe the mapping between the
+given coordinate variables and the true latitude and longitude coordinates, to describe the
 figure of the Earth used to define the latitude and longitude coordinates, or to describe another
-coordinate reference system definition used by some coordinates or auxiliary coordinates. Such a
-grid mapping variable provides the description of the mapping via a collection of attached
+coordinate reference system definition used by some coordinates or auxiliary coordinates.
+
+When the coordinate variables for a horizontal grid are not longitude and latitude, it is required that further information is provided to geo-locate the horizontal position.  A __grid mapping variable__ provides this information.
```

Comment:
I think the last sentence of this paragraph should say instead  "A __grid mapping variable__ and/or true latitude and longitude coordinates provide this information.".

This is to make it clear that is is still OK to provide just lats/lons (with no gridmapping), or both lats/lons and a grid mapping (as was the case pre v1.8).

Thanks

Reply:
@davidhassell 
This is the explicit intent of the very next sentence.  The sentence following this one makes it clear that it is fine to include longitudes and latitudes to assist in location the data in horizontal space.  It also mandates this practice if there is no grid_mapping supplied.

Does this meet your requirement that it should be clear that it is OK to provide just longitudes and latitudes with no grid_mapping?

Reply:
We actually have a deeper problem in the document. The text assumes that latitude and longitude coordinate values are somehow immutable 'true' values. This is a bad assumption. Latitude and longitude without a grid_mapping (and without a vertical datum / ellipsoid, which the grid_mapping doesn't really address) are untethered, approximate quantities. The ambiguity is ignorable for climate models and coarse resolution observation data, but not for imaging satellite or LIDAR data where cell sizes are on the order of 0.5 - 10 meters. This is part of the reason I suggest that we drop the word 'true' when referring to latitude and longitude.

We have such a long history of accepting this naïve expression of latitude and longitude that I don't think it's useful to try to force the use of a grid_mapping for latitude and longitude, and our grid_mapping attributes don't actually include sufficient information to fully solve the problem, but I'm strongly tempted to include verbiage pointing out the weakness.

Reply:
@JimBiardCics whilst I agree with the sentiment and detail of your comments here, I feel that you are raising a separate issue from the one under discussion in this PR.  May I ask that this topic be moved into a separate thread please?

Reply:
@marqh I was editorializing, and I'm OK with not tackling the whole issue here. I do think we should take advantage of this PR to remove the adjective 'true' from the descriptions of latitude and longitude variables.

Reply:
Not quite - I'd like it to be clear that is also OK to have *both* a grid mapping variable *and* lat/lon coordinate variables, hence the "and/or" in my original suggested text. The current text only mentions including lat/lon coordinates in the absence of a grid mapping variable. This is important for backwards compatibility.